scripts/update-abi-registry.js

```diff
@@ -17,6 +17,12 @@ async function fetchNodeVersions () {
   const schedule = await getJSONFromCDN('nodejs/Release/schedule.json')
   const versions = {}
 
+  const targetOverride = {
```

Comment:
Can you add a comment here that gives a description of what this variable is supposed to do, for future folks?

Reply:
Sure! Done.